### PR TITLE
Refactor change payments

### PR DESCRIPTION
**Pull request by edolyne**
@jbaxleyiii Here is the current work in progress that we had been chatting through.

### CODE REVIEW

**Comment by jbaxleyiii** [comment]
**[Storybook Hub](https://hub.getstorybook.io/)** is building storybooks for every commit in this PR.

**Latest Storybook: https://eb93cb9d-63c8-44bd-b153-c4ad57e5a81c.sbook.io/**
<sub>( Visit [here](https://hub.getstorybook.io/app/github.com/NewSpring/Holtzman/branch/refactor-changePayments/) for more info )</sub>

> _This comment is automatically added by Storybook Hub_

**Comment by edolyne** [comment]
@johnthepink @jbaxleyiii Could one of you guys take a look at this and make sure that I've done the refactor, and typing correctly here?  I'd like to make sure that there are things that I haven't missed in doing this.  The only place that I'm managing state is on the main index file.

File: imports/blocks/change-payments/Layout.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  changeAccounts,
+}: ILayout) => (
+  <div className="soft-double-top">
+    <div className="soft-sides flush-bottom push-double-top@lap-and-up">
+      <h4 className="text-center flush-bottom">
+        Change Payment Account
+      </h4>
+    </div>
+    <div className="soft">
+      { savedAccounts.map((account, key) => (
```
Comment (jbaxleyiii) [inline imports/blocks/change-payments/Layout.js]: extra whitespace here

File: imports/blocks/change-payments/paymentCard/index.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    </div>
+
+    <div className="locked-ends locked-sides">
+      <input
+        type="checkbox"
+        id={`label${accountId}`}
+        readOnly
+        checked={Number(accountId) === Number(selectedAccountId)}
+        style={{
```
Comment (jbaxleyiii) [inline imports/blocks/change-payments/paymentCard/index.js]: Let's move this style object to the top of the file and just refer to it here

File: imports/blocks/change-payments/paymentCard/index.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+          opacity: 0,
+          position: "absolute",
+          top: 0,
+          left: 0,
+          padding: "50px",
+        }}
+      />
+      <label
+        htmlFor={`label${accountId}`}
+        style={{
```
Comment (jbaxleyiii) [inline imports/blocks/change-payments/paymentCard/index.js]: same with this

**Comment by jbaxleyiii** [comment]
Also https://travis-ci.org/NewSpring/Holtzman/builds/174894802#L478 failing

File: imports/blocks/change-payments/index.js [hunk 4]
```diff
@@ -1,45 +1,52 @@
...
     this.props.dispatch(modal.hide());
   }
 
-  choose = (e) => {
+  choose = (e: SyntheticInputEvent) => {
     e.preventDefault();
 
     const { id } = e.currentTarget;
-    let act = {};
+    // XXX This is wrong, and needs to be changed...eventaully
```
Comment (jbaxleyiii) [inline imports/blocks/change-payments/index.js]: whats wrong here?

Comment (edolyne) [inline imports/blocks/change-payments/index.js]: @jbaxleyiii When working through this with @johnthepink we discussed that `any` isn't the best solution long term for the typed definition here.

Comment (jbaxleyiii) [inline imports/blocks/change-payments/index.js]: ah perfect! Could you clarify that?

`XXX 'any' is not a specific enough type`

**Comment by jbaxleyiii** [comment]
@edolyne great work!

**Pull request merged**